modules/ding_mkws/js/ding_mkws.theme.js

```diff
@@ -0,0 +1,352 @@
+        }
+        variables.items.push(out);
+      }
+    }
+
+    if (data.hits.length === 0) {
+      variables.title = "";
+    }
+
+    return variables;
+  };
+
+  $.templates("dingMkwsNodeWidget", "" +
+    "<div class='ding-mkws-widget ding-mkws-widget-node'>" +
+      "{{if title}}" +
+      "<div class='ding-mkws-title'>{{:title}}</div>" +
+      "{{/if}}"+
+      "<div class='ding-mkws-content'>" +
+        "<div class='ding-mkws-rows'>" +
+          "<div class='row'>" +
+            "{{if target}}" +
+              "<div class='left-label'>" +
+                "<p class='ding-mkws-target'>" +
+                  Drupal.t('Target') +
+                "</p>" +
+              "</div>" +
+            "{{/if}}" +
+            "{{if target}}" +
+            "<div class='right-value'>" +
+                "<p class='ding-mkws-target'>" +
+                  '{{:target}}' +
+                "</p>" +
+              "</div>" +
+            "{{/if}}" +
+          "</div>" +
+
+          "<div class='row'>" +
+            "{{if title}}" +
+              "<div class='left-label'>" +
+                "<p class='ding-mkws-title'>" +
+                  Drupal.t('Title') +
+                  "</p>" +
+              "</div>" +
+            "{{/if}}" +
+            "{{if title}}" +
+              "<div class='right-value'>" +
+                "{{if url}}" +
+                  "<a class='ding-mkws-title' href='{{:url}}' target='_blank'>" +
+                    '{{:title}}' +
+                  "</a>" +
+                  "{{else}}" +
+                    "<p class='ding-mkws-title'>" +
+                      '{{:title}}' +
+                    "</p>" +
+                "{{/if}}" +
+              "</div>" +
+            "{{/if}}" +
+          "</div>" +
+
+          "<div class='row'>" +
+            "{{if author}}" +
+              "<div class='left-label'>" +
+                "<p class='ding-mkws-author'>" +
+                  Drupal.t('Author') +
+                  "</p>" +
+              "</div>" +
+            "{{/if}}" +
+            "{{if author}}" +
+              "<div class='right-value'>" +
+                  "<p class='ding-mkws-author'>" +
+                    '{{:author}}' +
+                  "</p>" +
+              "</div>" +
+            "{{/if}}" +
+          "</div>" +
+
+          "<div class='row'>" +
+            "{{if date}}" +
+              "<div class='left-label'>" +
+                "<p class='ding-mkws-date'>" +
+                  Drupal.t('Date') +
+                "</p>" +
+              "</div>" +
+            "{{/if}}" +
+            "{{if date}}" +
+              "<div class='right-value'>"+
+                "<p class='ding-mkws-date'>" +
+                  '{{:date}}' +
+                "</p>" +
+              "</div>"+
+            "{{/if}}" +
+          "</div>" +
+        "</div>" +
+      "</div>" +
+      "<a class='ding-mkws-more-link' href='/search/meta/{{:more_link}}'>{{:more_text}}</a>" +
```

Comment:
href needs Drupal.settings.basePath + Drupal.settings.pathPrefix to work consistently